terraform/modules/tf_globals/alerts_firehose/iam.tf

```diff
@@ -1,8 +1,12 @@
 // IAM Role: Alert Firehose S3 Role
 resource "aws_iam_role" "firehose" {
-  name = "${var.prefix}_streamalert_delivery_firehose"
-
+  name               = "${var.prefix}_firehose_alert_delivery"
+  path               = "/streamalert/"
```

Comment:
Since you includes region in the path for the role created for kinesis, we probably also want region in the path here 
```
path               = "/streamalert/${var.region}/"
```

Reply:
I'm not sure I understand why you want this here??

Reply:
where is the role you're referring to with the region in it?

Reply:
Yeah, ignore me. I misread, you are right, it is for cloudwatch.

Reply:
👍